plugins/tradingAdvisor/baseTradingMethod.js

```diff
@@ -303,8 +312,13 @@ Base.prototype.advice = function(newPosition, _candle) {
     return;
 
   // ignore if advice equals previous advice
-  if(newPosition === this._prevAdvice)
+  // updated to take the current actual position into account in case a previous advice resulted in a cancelled/failed trade
+  // ** still need to consider a way to deal with an enhancement to not use the full balance on every trade
```

Comment:
can you remove this comment? Not using the full balance is an issue with a lot bigger scope than simply the execution logic.

Reply:
Done